csrc/u8x8_d_st7565.c

```diff
@@ -143,7 +143,7 @@ uint8_t u8x8_d_st7565_common(u8x8_t *u8x8, uint8_t msg, uint8_t arg_int, void *a
 	controller: It is not allowed to write beyond the display limits.
 	This is in fact an issue within flip mode.
       */
-      if ( c + x > 132u )
+      if ( ((uint16_t) (c + x)) > 132u )
```

Comment:
This point is more complicated. 
It is true and intended that there may happen an implicit conversion.
I do not know why there should be a warning. 
I have studied user manual for the XC8, but i do not know why the warning is there and how it should be fixed. (personal note: sizeof(int)==2 for XC8 according to chapter 5.4.2)
I would pefer a separate issue to improve this (if required).